src/test/java/de/doubleslash/keeptime/common/DateFormatterTest.java

```diff
@@ -38,4 +43,21 @@ void zeroSecondsBetweenTest() {
       final long secondsBewtweenSwitched = DateFormatter.getSecondsBewtween(endDate, startDate);
       assertThat(secondsBewtweenSwitched, Matchers.is(0l)); // why??
    }
+
+   @Test
+   void applySystemLocaleShouldSetDatePickerLocale() {
+      // GIVEN
+      Locale.setDefault(Locale.ENGLISH);
+      DateFormatter.setSystemLocale(Locale.GERMAN);
+      DatePicker datePicker = new DatePicker();
+      LocalDate date = LocalDate.of(2024, 6, 3);
+
+      // WHEN
+      DateFormatter.applySystemLocaleOnDate(datePicker);
+      datePicker.setValue(date);
+
+      // THEN
+      String shown = datePicker.getEditor().getText();
+      assertTrue(shown.contains("Montag") || shown.contains("06.2024") || shown.contains("06"));
```

Comment:
assert is weird with 3 items in OR condition -> only one must be true?
Better create three asserts with each aspect.